lib/sogou/search/api/service/report.rb

```diff
@@ -9,32 +9,32 @@ def initialize
             super('http://api.agent.sogou.com:8080/sem/sms/v1/ReportService?wsdl')
           end
 
-          def get_report_id(start_date, end_date, type, fields, options={})
-            operation = client.operation(:get_report_id)
-            resp = client.call(:get_report_id, message: {
-              report_request_type: {
-                performance_data: fields,
-                start_date: start_date,
-                end_date: end_date,
-                report_type: type
-              }
-            })
-            puts resp.header
-            resp.body
+          def get_report_id(start_date, end_date, type, fields, options = {}, &block)
+            command = make_command(:get_report_id,
```

Comment:
if you put the first parameter on the next line you wouldn't have to do such indentation :)

Reply:
hmm, it doesn't look pretty :(